plugins/MumblePlugin.h

```diff
@@ -1608,7 +1644,65 @@ struct MUMBLE_API_STRUCT_NAME {
 																				 mumble_connection_t connection,
 																				 const char *comment);
 
+#	if SELECTED_API_VERSION >= MUMBLE_PLUGIN_VERSION_CHECK(1, 3, 0)
 
+	/**
+	 *
+	 * Requests Mumble to link the given channel to the given linkID.
+	 *
+	 * @param callerID The ID of the plugin calling this function
+	 * @param channelID The ID of the channel to link
+	 * @param linkID The ID of the channel to link to channelID
+	 * @returns The error code. If everything went well, STATUS_OK will be returned.
+	 */
+	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *requestLinkChannel)(mumble_plugin_id_t callerID,
+																		 mumble_channelid_t channelID,
+																		 mumble_channelid_t linkID);
+
+	/**
+	 *
+	 * Requests Mumble to unlink the given channel from the given unlinkID.
+	 *
+	 * @param callerID The ID of the plugin calling this function
+	 * @param channelID The ID of the channel to unlink
+	 * @param unlinkID The ID of the channel to remove link from channelID
+	 * @returns The error code. If everything went well, STATUS_OK will be returned.
+	 */
+	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *requestUnlinkChannel)(mumble_plugin_id_t callerID,
+																		   mumble_channelid_t channelID,
+																		   mumble_channelid_t unlinkID);
+
+	/**
+	 *
+	 * Requests Mumble to link the given channels to the given channel.
+	 *
+	 * @param callerID The ID of the plugin calling this function
+	 * @param channelID The ID of the channel to link
+	 * @param linkList The list of channel IDs to link to the channelID
+	 * @param linkCount The number of elements in the link list
+	 * @returns The error code. If everything went well, STATUS_OK will be returned.
+	 */
+	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *requestLinkChannels)(mumble_plugin_id_t callerID,
+																		  mumble_channelid_t channelID,
+																		  mumble_channelid_t *linkList,
```

Comment:
```suggestion
																		  mumble_channelid_t *channels,
```
Linking is transitive - that is if channel A is linked to B and B is linked to C then A is also linked to C. Therefore, it is sufficient to specify the group of channels that shall be linked together